<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>searchInfoLessons</class>
 <widget class="QWidget" name="searchInfoLessons">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1065</width>
    <height>727</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QTableView" name="tableView">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>761</width>
     <height>541</height>
    </rect>
   </property>
  </widget>
  <widget class="QWidget" name="widget" native="true">
   <property name="geometry">
    <rect>
     <x>820</x>
     <y>120</y>
     <width>171</width>
     <height>421</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QPushButton" name="pushButton">
      <property name="styleSheet">
       <string notr="true">font: 9pt &quot;黑体&quot;;</string>
      </property>
      <property name="text">
       <string>提交信息修改</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="pushButton_2">
      <property name="styleSheet">
       <string notr="true">font: 9pt &quot;黑体&quot;;</string>
      </property>
      <property name="text">
       <string>撤销信息修改</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="pushButton_3">
      <property name="styleSheet">
       <string notr="true">font: 9pt &quot;黑体&quot;;</string>
      </property>
      <property name="text">
       <string>添加信息</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="pushButton_4">
      <property name="styleSheet">
       <string notr="true">font: 9pt &quot;黑体&quot;;</string>
      </property>
      <property name="text">
       <string>删除选中课程</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="pushButton_7">
      <property name="styleSheet">
       <string notr="true">font: 9pt &quot;黑体&quot;;</string>
      </property>
      <property name="text">
       <string>查看选该课程的人</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="pushButton_9">
      <property name="styleSheet">
       <string notr="true">font: 9pt &quot;黑体&quot;;</string>
      </property>
      <property name="text">
       <string>提交所有课程成绩</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="pushButton_13">
      <property name="styleSheet">
       <string notr="true">font: 9pt &quot;黑体&quot;;</string>
      </property>
      <property name="text">
       <string>成绩降序排列</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="widget_2" native="true">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>580</y>
     <width>431</width>
     <height>111</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="2">
     <widget class="QPushButton" name="pushButton_5">
      <property name="styleSheet">
       <string notr="true">font: 9pt &quot;黑体&quot;;</string>
      </property>
      <property name="text">
       <string>查询</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QLineEdit" name="lineEdit_search_number"/>
    </item>
    <item row="2" column="1">
     <widget class="QPushButton" name="pushButton_8">
      <property name="styleSheet">
       <string notr="true">font: 9pt &quot;黑体&quot;;</string>
      </property>
      <property name="text">
       <string>显示/更新所有课程信息</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QLineEdit" name="lineEdit_search_name"/>
    </item>
    <item row="1" column="2">
     <widget class="QPushButton" name="pushButton_6">
      <property name="styleSheet">
       <string notr="true">font: 9pt &quot;黑体&quot;;</string>
      </property>
      <property name="text">
       <string>查询</string>
      </property>
     </widget>
    </item>
    <item row="0" column="0">
     <widget class="QPushButton" name="pushButton_10">
      <property name="styleSheet">
       <string notr="true">font: 9pt &quot;黑体&quot;;</string>
      </property>
      <property name="text">
       <string>课程名:</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QPushButton" name="pushButton_11">
      <property name="styleSheet">
       <string notr="true">font: 9pt &quot;黑体&quot;;</string>
      </property>
      <property name="text">
       <string>课序号:</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1065</width>
     <height>727</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QWidget" name="widget_4" native="true">
   <property name="geometry">
    <rect>
     <x>600</x>
     <y>590</y>
     <width>321</width>
     <height>50</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QComboBox" name="comboBox_depart"/>
    </item>
    <item>
     <widget class="QPushButton" name="pushButton_12">
      <property name="styleSheet">
       <string notr="true">font: 9pt &quot;黑体&quot;;</string>
      </property>
      <property name="text">
       <string>查询某系开课</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="widget_average" native="true">
   <property name="geometry">
    <rect>
     <x>850</x>
     <y>30</y>
     <width>193</width>
     <height>78</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <widget class="QPushButton" name="pushButton_14">
      <property name="styleSheet">
       <string notr="true">font: 9pt &quot;黑体&quot;;</string>
      </property>
      <property name="text">
       <string>平均分</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="lineEdit_average"/>
    </item>
   </layout>
  </widget>
  <zorder>label_3</zorder>
  <zorder>tableView</zorder>
  <zorder>widget</zorder>
  <zorder>widget_2</zorder>
  <zorder>widget_4</zorder>
  <zorder>widget_average</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
